Source/WebKit/UIProcess/Cocoa/WebProcessProxyCocoa.mm

```diff
@@ -259,5 +259,37 @@
 }
 #endif
 
+#if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
+void WebProcessProxy::logOnBehalfOfWebContent(std::span<const char> logSubsystemIncludingNullTerminator, std::span<const char> logCategoryIncludingNullTerminator, std::span<const char> logStringIncludingNullTerminator, uint8_t logType, int32_t pid)
```

Comment:
If we are going to be logging from the UIProcess, then this should occur on a MessageReceiver that does not run on the main thread.

Reply:
Has this been addressed? It is not obvious to me that this code is running on a background thread. If it is, we should add an assertion.